### PR TITLE
Use event handler on form submission

### LINKED ISSUE #1112: media:group:create
**Describe the bug**
When creating a group if you use the enter key command it will reload the app.

**To Reproduce**
1. Go to /media
2. Click on "Create Sub Group"
3. Type in new group name
4. Press "enter" key
5. See error

**Expected behavior**
App should not reload and pressing enter should create the new group.

**Screenshots**
image

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

### PR DESCRIPTION
**Pull request by rjpals**
Closes #1112

### CODE REVIEW

**Comment by agalin920** [comment]
The button has an onClick handler that also calls handleCreateGroup. A better solution would be to give the button type of 'submit'  and remove its onClick handler so only the onSubmit handler gets fired

**Review by agalin920 — CHANGES_REQUESTED** [review]
The button has an onClick handler that also calls handleCreateGroup. A better solution would be to give the button type of 'submit' and remove its onClick handler so only the onSubmit handler gets fired

File: src/apps/media/src/app/components/MediaCreateGroupModal.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ export function MediaCreateGroupModal(props) {
           <Button
             variant="contained"
             color="success"
-            onClick={handleCreateGroup}
+            type="save"
```
Comment (agalin920) [inline src/apps/media/src/app/components/MediaCreateGroupModal.js:48]: Why is this being added?

Comment (rjpals) [reply src/apps/media/src/app/components/MediaCreateGroupModal.js]: This was a typo, my bad. Interestingly, it did actually fix the issue tho.

**Comment by agalin920** [comment]
@rjpals MediaEditGroupModal has the same issue could you also include it in this PR

**Review by shrunyan — APPROVED** [review]

**Pull request merged**